docs/en/administrator-guide/query_cache.md

```diff
@@ -0,0 +1,190 @@
+---
+{
+    "title": "QUERY CACHE",
+    "language": "en"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# QUERY CACHE
+
+
+## 一 Demond
+
+Usually, we have cache in the corresponding database layer which is focus on the query content.However the data granularity in the cache is too slow and it would have effect only when  data in the table didn't change.Obviously, it could not reduce the heavy IO load from the CRUD of the business system. So it comes the database caching technology, which could highspeed caching the hot data, improve the response speed of the application and greatly relieve the load of the backend database.
```

Comment:
*However the data granularity in the cache is too **slow** and ...*

Is it *slow* or *small*?

Reply:
the data granularity in the cache is too slow